boot-core/tests/common/mod.rs

```diff
@@ -29,21 +63,24 @@ fn docker_container_setup() {
         env::set_var("LOCAL_MNEMONIC", LOCAL_MNEMONIC);
     }
 
-    log::info!("using RUST_LOG: {}", env::var("RUST_LOG").unwrap());
-    log::info!("using STATE_FILE: {}", env::var("STATE_FILE").unwrap());
+    log::info!("Using RUST_LOG: {}", env::var("RUST_LOG").unwrap());
+    log::info!("Using CONTAINER_NAME: {}", container);
+    log::info!("Using STATE_FILE: {}", env::var("STATE_FILE").unwrap());
     log::info!(
-        "using LOCAL_MNEMONIC: {}",
+        "Using LOCAL_MNEMONIC: {}",
         env::var("LOCAL_MNEMONIC").unwrap()
     );
 
     if !*init {
+        ensure_container_removal();
```

Comment:
If you read the `CONTAINER_NAME` above you should pass it in here instead of re-reading it in the function.